Makefile

```diff
@@ -66,21 +66,9 @@ test_send_mail_quiet: check_install $(TEST_DEP) install
 	rm -f $(INSTALL)/data/send_mail/task_sched_disable_alerts
 	perl $(INSTALL_BIN)/task_schedule.pl -alert $(USER) -config $(INSTALL)/data/send_mail/send_mail.config -fast 5 
 
-install:
-	mkdir -p $(INSTALL_BIN); rsync --times --cvs-exclude $(BIN) $(INSTALL_BIN)/
-	mkdir -p $(INSTALL_DOC)
-	pod2html task_schedule.pl > $(INSTALL_DOC)/index.html
-	rm -f pod2htm?.tmp
-
-# Install task_schedule.pl using "skare3" convention: put into SKA_ARCH_OS/bin
-# and launched with /usr/bin/env perl instead of hardwired /usr/bin/env /proj/sot/ska/bin/perl
 install3:
-	echo '#!/usr/bin/env perl' > task_schedule3.pl
-	tail -n +2 task_schedule.pl >> task_schedule3.pl
-	sed s/watch_cron_logs\.pl/watch_cron_logs3.pl/ -i task_schedule3.pl
-	chmod +x task_schedule3.pl
 	cp -p task_schedule3.pl $(SKA_ARCH_OS)/bin/
-	rm task_schedule3.pl
+	chmod +x $(SKA_ARCH_OS)/bin/task_schedule3.pl
```

Comment:
`SKA_ARCH_OS` is deprecated in favor of `SKA_ROOT`.

Reply:
(But this will work, it's just not preferred for new-ish stuff).

Reply:
OK.

Reply:
I can probably cut the chmod anyway, as it isn't a newly created file anymore and should keep the executable status from the repo.

Reply:
Agreed on cutting.

Reply:
Also, if `SKA_ARCH_OS` is deprecated, where is `SKA_ROOT` set?  If that's what you want, we should probably add it to `flt_envs` (I don't see it).  INSTALL_BIN would probably work here too as we're still importing the Makefile.FLIGHT.

Reply:
I set the Makefile basically back to the original install target, as it makes the most sense to do the conda build with just a cp at this point, now that task_schedule3.pl is already "built".